test/locale/la.test.js

```diff
@@ -0,0 +1,183 @@
+    [10, 'october', 'octobris', 'oct.'],
+    [11, 'november', 'novembris', 'nov.'],
+    [12, 'december', 'decembris', 'dec.']
+  ]
+
+  it('in genitive case if day of month is output', () => {
+    T.forEach((t) => {
+      const dayjsDay = dayjs(['2020-', t[0], '-01'].join('')).locale('la').format('D MMMM')
+      const expected = ['1 ', t[2]].join('')
+      expect(dayjsDay).toBe(expected)
+    })
+  })
+
+  it('in nominative if day of month is not output', () => {
+    T.forEach((t) => {
+      const dayjsDay = dayjs(['2020-', t[0], '-01'].join('')).locale('la').format('MMMM')
+      const expected = t[1]
+      expect(dayjsDay).toBe(expected)
+    })
+  })
+
+  it('also as a shortcut', () => {
+    T.forEach((t) => {
+      const dayjsDay = dayjs(['2020-', t[0], '-01'].join('')).locale('la').format('MMM')
+      const expected = t[3]
+      expect(dayjsDay).toBe(expected)
+    })
+  })
+})
+
+it('Latin: Test relative time', () => {
+  const T = [
+    [44.4, 'second', 'paucæ secundæ'], // a few seconds
+    [89.5, 'second', 'minuta'], // a minute
+    [2, 'minute', '2 minutæ'], // 2 minutes
+    [43, 'minute', '43 minutæ'], // 43 minutes
+    [45, 'minute', 'hora'], // an hour
+    [3, 'hour', '3 horæ'], // 3 hours
+    [21, 'hour', '21 horæ'], // 21 hours
+    [1, 'day', 'dies'], // a day
+    [3, 'day', '3 dies'], // 3 day
+    [25, 'day', '25 dies'], // 25 days
+    [1, 'month', 'mensis'], // a month
+    [2, 'month', '2 menses'], // 2 month
+    [10, 'month', '10 menses'], // 10 month
+    [1, 'year', 'annus'], // a year
+    [2, 'year', '2 anni'], // 2 year
+    [5, 'year', '5 anni'], // 5 year
+    [18, 'month', '2 anni'] // 2 years
+  ]
+
+  T.forEach((t) => {
+    // dayjs.locale('la')
+    const dayjsDay = dayjs()
+    const dayjsCompare = dayjs().add(t[0], t[1])
+    const expectedFrom = ['abhinc ', t[2]].join('')
+    const expectedTo = ['ad ', t[2]].join('')
+    const expectedFromTrue = t[2]
+
+    expect(dayjsDay.from(dayjsCompare)).toBe(expectedFrom)
+    expect(dayjsDay.to(dayjsCompare)).toBe(expectedTo)
+    expect(dayjsDay.from(dayjsCompare, true)).toBe(expectedFromTrue)
+  })
+})
+
+it('Latin: Test ordinal numbers', () => {
+  const T = [
+    [1, 'Iᵒ'],
+    [2, 'IIᵒ'],
+    [3, 'IIIᵒ'],
+    [4, 'IVᵒ'],
+    [5, 'Vᵒ'],
+    [6, 'VIᵒ'],
+    [7, 'VIIᵒ'],
+    [8, 'VIIIᵒ'],
+    [9, 'IXᵒ'],
+    [10, 'Xᵒ'],
+    [20, 'XXᵒ'],
+    [30, 'XXXᵒ'],
+    [31, 'XXXIᵒ']
+    // [10, 'Xᵒ'],
+    // [20, 'XXᵒ'],
+    // [40, 'XLᵒ'],
+    // [50, 'Lᵒ'],
+    // [90, 'XCᵒ'],
+    // [100, 'Cᵒ'],
+    // [200, 'CCᵒ'],
+    // [500, 'Dᵒ'],
+    // [900, 'CMᵒ'],
+    // [1000, 'Mᵒ'],
+    // [2000, 'MMᵒ']
+  ]
+
+  T.forEach((t) => {
+    const dayjsDay = dayjs(['2020-01-', t[0]].join('')).locale('la').format('Do')
+    const expected = t[1]
+    expect(dayjsDay).toBe(expected)
+  })
+})
+
+describe('Latin: Test weeday names:', () => {
+  const T = [
+    ['2020-01-05', 'Dominica', 'Dominica', 'Dom.'],
```

Comment:
Well, I usually see _Dominca_ and _Sabbato_ written with capital first letter (former probably because it relates to the Lord), while _feria_ is not a proper word. Nevertheless, I need to verify this usage either in _Missale Romanum_ (ed. typ tertia) or by an Ecclesiastic Latin speaker e.g. a Church lawyer).

Reply:
Okay, I’ll lowercase all weekday names. (src: [_Missale Romanum_ (1977), ed. typ. altera](https://drive.google.com/file/d/1w8sX42WQnbfJQ9Vm7nVyUAbtcJ32AS9f/view?usp=sharing))